Alignment/HIPAlignmentAlgorithm/src/HIPAlignmentAlgorithm.cc

```diff
@@ -107,16 +123,55 @@ HIPAlignmentAlgorithm::initialize( const edm::EventSetup& setup,
 
   edm::ESHandle<Alignments> globalPositionRcd;
   // FIXME! temporary solution to get highest possible run number
-  const unsigned int MAX_VAL(std::numeric_limits<unsigned int>::max());
-  edm::ValidityInterval iov(setup.get<GlobalPositionRcd>().validityInterval() );
-  if (iov.first().eventID().run()!=1 || iov.last().eventID().run()!=MAX_VAL) {
-    throw cms::Exception("DatabaseError")
-      << "@SUB=AlignmentProducer::applyDB"
-      << "\nTrying to apply "<< setup.get<GlobalPositionRcd>().key().name()
-      << " with multiple IOVs in tag.\n"
-      << "Validity range is "
-      << iov.first().eventID().run() << " - " << iov.last().eventID().run();
-  }
+//  const unsigned int MAX_VAL(std::numeric_limits<unsigned int>::max());
+ // edm::ValidityInterval iov(setup.get<GlobalPositionRcd>().validityInterval() );
+//  if (iov.first().eventID().run()!=1 || iov.last().eventID().run()!=MAX_VAL) {
+//    throw cms::Exception("DatabaseError")
+//      << "@SUB=AlignmentProducer::applyDB"
+//      << "\nTrying to apply "<< setup.get<GlobalPositionRcd>().key().name()
+//      << " with multiple IOVs in tag.\n"
+//      << "Validity range is "
+//      << iov.first().eventID().run() << " - " << iov.last().eventID().run();
+//  }
+
+//  const Rcd & record = setup.get<TrackerAlignmentRcd>();
+  const edm::ValidityInterval & validity = setup.get<TrackerAlignmentRcd>().validityInterval();
+  const edm::IOVSyncValue first1 = validity.first();
+	std::cout << "xiaomeng "<<first1.eventID().run()<<std::endl;
```

Comment:
@meng-xiao would you please replace `cout` with usage of the messageLogger service?